recipes/ftxui/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+import conan.tools.files
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.43.0"
+
+
+class FTXUIConan(ConanFile):
+    name = "ftxui"
+    description = "C++ Functional Terminal User Interface."
+    license = "MIT"
+    topics = ("ncurses", "terminal", "screen", "tui")
+    homepage = "https://github.com/ArthurSonzogni/FTXUI"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def _validate_compiler_settings(self):
+        compiler = self.settings.compiler
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == 'gcc' and version < '8':
+            raise ConanInvalidConfiguration("gcc 8 required")
+        if compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "17")
+
+    def validate(self):
+        self._validate_compiler_settings()
+        if self.settings.compiler in ["Visual Studio", "msvc"]:
+            raise ConanInvalidConfiguration("Visual Studio unsupported")
```

Comment:
Seems to be supported. I guess it's more OS related than compiler related.

Reply:
As I previously said, the support is experimental per readme, and I'm having hard time verifying it without windows setup. I can try and remove it, if it just works - let it be so.

Reply:
fixed

Reply:
Since it seems you're helping me testing it, let's make sure it works, and allow compiling it. Merged your fixes.